resume.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>wommy.io - resume</title>
+
+		<link rel='stylesheet' href='style.css'>
+		<style>
+			#main {
+				margin: 1rem;
+			}
+			h2 {
+				border-bottom: 1px solid silver;
+				padding-bottom: .3rem;
+			}
+			.margin{
+				margin: 1rem 0;
+			}
+			.indent{
+				padding-left: 1.25rem;
+			}
+			.highlight{
+				background-color: gainsboro;
+				border-radius: 3px;
+				padding: .2em .4em;
+				font-size: 85%;
+				font-family: monospace;
+			}
+		</style>
+	</head>
+	<body id='resume'>
+		<nav>
+			<a href='./index.html'>home</a>
+		</nav>
+		<div id='main'>
+			<h1 style="display:none;">resume</h1>
+			<div id='content'>
+				<h2 style="display:none;">contact</h2>
```

Comment:
Same here

Reply:
same thing: i had a title for each section, but this one interrupted the visual flow so i removed it

open to suggestions

Reply:
I don't think the design is a huge concern at this point, so whatever you think is right!